src/libstd/sys/windows/mod.rs

```diff
@@ -151,6 +152,27 @@ fn cvt<I: PartialEq + Zero>(i: I) -> io::Result<I> {
     }
 }
 
+fn dur2timeout(dur: Duration) -> libc::DWORD {
+    // Note that a duration is a (u64, u32) (seconds, nanoseconds) pair, and the
```

Comment:
I’m implying that there should eventually exist a method to do such conversion on the `Duration` itself and that internal `Duration` representation shouldn’t be really relied on the end-user side (even if through the two methods).